tsparser/src/parser/resources/apis/api.rs

```diff
@@ -194,14 +203,44 @@ pub const ENDPOINT_PARSER: ResourceParser = ResourceParser {
                         None => None,
                         Some(t) => Some(pass.type_checker.resolve_type(module.clone(), &t)),
                     };
-                    (request, response)
+
+                    describe_endpoint(pass.type_checker, methods, path, request, response, raw)?
+                }
+                EndpointKind::TypedStream {
```

Comment:
Maybe better to move this match case below `EndpointKind::Raw`, since that one is simpler and more closely aligned with `EndpointKind::Typed`